app/controllers/users_controller.rb

```diff
@@ -5,5 +5,7 @@ def index
 
   def show
     @user = User.find(params[:id])
+    @posts = Post.where(author_id: params[:id])
+    @post = Post.find(params[:id])
```

Comment:
- [ ] Good job so far. but you still have some issues that need to be fixed. When I click to show the user I got an error so then I can't create a post also I can't add a comment or like. Please kindly fix that 🙃 🙃 

<img width="815" alt="Screen Shot 2023-01-28 at 15 46 23" src="https://user-images.githubusercontent.com/74501523/215268171-09278751-1e36-4fda-9dc0-579fef1877c2.png">